app/mempool/lane.go

```diff
@@ -31,6 +31,11 @@ type Lane struct {
 	// txIndex holds the uppercase hex-encoded hash for every transaction
 	// currently in this lane's mempool.
 	txIndex map[string]struct{}
+
+	// OnFilled is a callback function that is called when the lane exceeds its limit.
+	OnFilled func(isFilled bool)
```

Comment:
OnFilled vs onLaneLimitReached or OnLaneLimitExceeded (reduce the variable isFilled)? can resolve na if as-is is ok

Reply:
agreed, prefer onLaneLimitReached or onLaneLimitExceed